### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by igor-stasiv**
[DEMO LINK](https://igor-stasiv.github.io/react_decompose/)

### CODE REVIEW

**Review by Burunduk003 — APPROVED** [review]
Good job! You may do "main tag" as a separate component as well, but I suppose your solution also fits the task.

File: src/components/Header/Header.jsx [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+function Header() {
+  return (
+    <header className="header">
+      <h1 className="header__title">Site Name</h1>
+      <nav className="navigation">
+        <a className="navigation__link" href="#about">About</a>
+        <a className="navigation__link" href="#services">Services</a>
+        <a className="navigation__link" href="#contact">Contact</a>
+      </nav>
```
Comment (VinnieJ-2k20) [inline src/components/Header/Header.jsx:12]: Let's move `Navigation` to separate component.

**Review by VinnieJ-2k20 — CHANGES_REQUESTED** [review]
Nice solution.
Just one edit needed.

**Review by VinnieJ-2k20 — APPROVED** [review]
Good job 👍

**Pull request closed without merging**